pages/_app.tsx

```diff
@@ -1,19 +1,22 @@
 import type { AppProps } from "next/app"
+import { motion, AnimatePresence } from "framer-motion"
 import localFont from "@next/font/local"
 import "../styles/globals.scss"
 
 export const ptSans = localFont({ src: "../public/PTSans-Regular.woff2", weight: "400", style: "normal", fallback: ["Trebuchet MS", "Geneva", "Helvetica Neue", "sans-serif"] })
 export const cardo = localFont({ src: "../public/Cardo-Regular.woff2", weight: "400", style: "normal", fallback: ["Times", "Times New Roman", "serif"] })
 
-export default function App({ Component, pageProps }: AppProps) {
+export default function App({ Component, pageProps, router }: AppProps) {
   return (
     <>
       <style jsx global>{`
         html {
           font-family: ${ptSans.style.fontFamily};
         }
       `}</style>
-      <Component {...pageProps} />
+      <AnimatePresence>
+        <Component {...pageProps} key={router.asPath} />
```

Comment:
This change to add the `key` prop that changes when routes change is the key to triggering page transitions, since changes in React component keys trigger re-renders that AnimatePresence will pick up on. Magic